### PR TITLE
script inputs

### PR DESCRIPTION
**Pull request by addshore**
- getData, empty job queue
- Allow input to scripts

### CODE REVIEW

File: getData.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
-WIKI_DOMAIN="addshore-alpha.wiki.opencura.com"
+WIKI_DOMAIN=$1
 
-# TODO empty the job queue before grabbing the databases!!!
+echo Getting data for $1
```
Comment (outdooracorn) [inline getData.sh:3]: Should we use quotes?
```suggestion
echo "Getting data for $1"
```

File: putData.sh [hunk 2]
```diff
@@ -1,9 +1,13 @@
 #!/bin/bash
 set -e
 
+echo Putting data for $1 to wikibase.dev
```
Comment (outdooracorn) [inline putData.sh:4]: Should we use quotes?
```suggestion
echo "Putting data for $1 to wikibase.dev"
```

**Comment by addshore** [comment]
Merging as I need it :)

File: putData.sh [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
+
 # Inputs for the script
+# TODO change wikibase.dev for real migration
+# TODO change cluster name for real migration
 NEW_PLATFORM_FREE_DOMAIN_SUFFIX="wikibase.dev"
```
Comment (outdooracorn) [inline putData.sh:9]: Do we want this to be a second (possibly optional) parameter? Or we just hard code change it when we come to do the migration to wikibase.cloud?

**Review by outdooracorn — APPROVED** [review]

**Pull request merged**